sonic_platform_base/sonic_sfp/sfputilhelper.py

```diff
@@ -123,7 +123,7 @@ def read_porttab_mappings(self, porttabfile):
     def get_physical_to_logical(self, port_num):
         """Returns list of logical ports for the given physical port"""
 
-        return self.physical_to_logical[port_num]
+        return self.physical_to_logical.get(port_num)
```

Comment:
What's the benefit of this approach? But if we change here, we should change in other places as well..

Reply:
Please take a look on these files:
+ https://github.com/Azure/sonic-platform-daemons/blob/master/sonic-xcvrd/scripts/xcvrd#L906
We expect for get_physical_to_logical to return None in the case of unknown port.
+ https://github.com/Azure/sonic-platform-common/blob/master/sonic_platform_base/sonic_sfp/sfputilbase.py#L603
That's SfpUtilBase implementation of the interface used in xcvrd 
No special benefit, just a fix.

Reply:
I guess the by other places you mean get_logical_to_physical - no reason for change if everything conforms the usage)

Reply:
Good. Please go ahead and create PR in community with this explanation.